### PR TITLE
Uploading  RT ENV for Druid and Struts

### PR DESCRIPTION
**Pull request by Aditi8007**

### CODE REVIEW

File: dubbo/0.0.1/besman-dubbo-RT-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# The variables that start with BESMAN_ are converted to environment vars.
+# If you wish to add any other vars that should be used globally, add the var using the below format.
+# BESMAN_<var name>: <value>
+# If you are not using any particular value, remove it or comment it(#).
+#*** - These variables should not be removed, nor left empty.
+# BESMAN_ORG - used to mention where you should clone the repo from, default value is Be-Secure
+BESMAN_ORG: Be-Secure #***
+
+# BESMAN_ARTIFACT_TYPE - project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: # project/ml model/training dataset #***
```
Comment (asa1997) [inline dubbo/0.0.1/besman-dubbo-RT-env-config.yaml:13]: Missing value

File: struts/0.0.1/besman-struts-RT-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# The variables that start with BESMAN_ are converted to environment vars.
+# If you wish to add any other vars that should be used globally, add the var using the below format.
+# BESMAN_<var name>: <value>
+# If you are not using any particular value, remove it or comment it(#).
+#*** - These variables should not be removed, nor left empty.
+# BESMAN_ORG - used to mention where you should clone the repo from, default value is Be-Secure
+BESMAN_ORG: Be-Secure #***
+
+# BESMAN_ARTIFACT_TYPE - project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: # project/ml model/training dataset #***
```
Comment (asa1997) [inline struts/0.0.1/besman-struts-RT-env-config.yaml:13]: Missing value

**Review by asa1997 — CHANGES_REQUESTED** [review]
Please update the list of environments here as well https://github.com/Be-Secure/besecure-ce-env-repo/tree/develop?tab=readme-ov-file#available-environments

**Comment by Aditi8007** [comment]
Need to raise the PR on develop branch.

**Pull request closed without merging**